main/webapp/modules/core/langs/translation-en.json

```diff
@@ -409,7 +409,7 @@
     "core-recon/error-contact": "Error contacting recon service",
     "core-recon/wd-recon-lang": "en",
     "core-recon/recon-col": "Reconcile column",
-    "core-recon/pick-service": "Pick a service or extension on left",
+    "core-recon/pick-service": "Pick a service or extension on left.",
```

Comment:
I would leave this message untouched since it is used in a dialog that is unrelated to the problem we are trying to solve.

Reply:
The message is untouched. The deletion is the message I had included earlier before the first review.